curriculum/js2/4.js

```diff
@@ -5,7 +5,11 @@
  */
 
 const solution = (a) => {
-  return 0;
+  if (a.length === 0) { return '' }
```

Comment:
you should return a number, not a string. so if length === 0, return 0. Your reduce would take care of it.